plextraktsync/media.py

```diff
@@ -148,13 +153,22 @@ def watched_on_trakt(self):
             self.show_trakt_id, self.season_number, self.episode_number
         )
 
-    def mark_watched_trakt(self):
+    def mark_watched_trakt(self, dry_run):
         if self.is_movie:
-            self.trakt_api.mark_watched(self.trakt, self.plex.seen_date)
+            if not dry_run:
+                logger.info(f"Marking as watched in Trakt: {self}")
+                self.trakt_api.mark_watched(self.trakt, self.plex.seen_date)
         elif self.is_episode:
-            self.trakt_api.mark_watched(
-                self.trakt, self.plex.seen_date, self.show_trakt_id
-            )
+            if self.show_reset_at and self.plex.seen_date.replace(tzinfo=None) < self.show_reset_at:
+                if not dry_run:
+                    logger.info(f"Show {self.plex.item.show().title} has been reset in trakt at {self.show_reset_at}.")
+                    logger.info(f"Marking {self.plex.item.show().title} as unwatched in Plex.")
+                    self.plex_api.reset_show(show=self.plex.item.show(), reset_date=self.show_reset_at)
+            elif not dry_run:
+                logger.info(f"Marking as watched in Trakt: {self}")
```

Comment:
dry_run should still print what it will do, that's the point of dry run to see what would happen

Reply:
> dry_run should still print what it will do

i agree. That's what i wrote first, then i saw some dry_run not printing in logs so i did the same